docs/webdriver-vs-protractor.md

```diff
@@ -27,3 +27,31 @@ tests.
 
 If you need the vanilla WebDriver `browser` object, you can access it via
 `browser.driver`
+
+Chaining WebDriver Actions
```

Comment:
Can we send readers to http://angular.github.io/protractor/#/locators instead, and maybe add a note about chaining actions there?